src/inausoft.netCLI/Commands/HelpCommandHandler.cs

```diff
@@ -45,23 +43,29 @@ public override int Run(HelpCommand command)
             }
             else
             {
-                var commandHandler = _commandHandlers.FirstOrDefault(it =>
-                    Attribute.IsDefined(it.GetCommandType(), typeof(CommandAttribute)) &&
-                    (Attribute.GetCustomAttribute(it.GetCommandType(), typeof(CommandAttribute)) as CommandAttribute).Name == command.SpecifiedCommandName);
+                var commandType = _configuration.CommandTypes.FirstOrDefault(it =>
+                    Attribute.IsDefined(it, typeof(CommandAttribute)) &&
+                    (Attribute.GetCustomAttribute(it, typeof(CommandAttribute)) as CommandAttribute).Name == command.SpecifiedCommandName);
 
-                if (commandHandler == null)
+                if (commandType == null)
```

Comment:
Lack of  test of `HelpCommandHandler`

Reply:
How would you test it?

Also - please do add yourself into README.md file as contributor - so that you would be visible on the main page. After all you do contributing to the solution by reviewing this code :)